sensors/01-meteorological-data.Rmd

```diff
@@ -145,14 +133,14 @@ into the command line. This uses the URL from above and the new file is named
 spectra.json.
 
 ```{sh eval=FALSE}
-curl -o spectra.json -X GET https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31
+curl -o spectra.json -X GET https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-01&until=2017-12-31
 ```
 
 #### Using R
 
 The following code sets the defaults for showing R code. 
 ```{r met-setup}
-knitr::opts_chunk$set(cache = TRUE, message = FALSE)
+knitr::opts_chunk$set(cache = FALSE, message = FALSE)
```

Comment:
Not sure why, but when cache=TRUE, I get this error:


> Quitting from lines 250-254 (_main.Rmd) 
> Error: lexical error: invalid char in json text.
>                                        https://terraref.ncsa.illinois.
>                     (right here) ------^

At this line: (252 in original PR) 

```r
datasets <- fromJSON(paste0(api_url, '/spaces/', spaces$id, '/datasets'))
```

Reply:
I've gotten that also, I delete all the files generated when I knit the .Rmd (_cache folder, _files folder, and .html file) and it works. I don't know why that happens.